HomeworkAnswers.md

```diff
@@ -0,0 +1,12 @@
+# Dice Coefficient Output
+## Experiment 1 (smoothing FWHM = 6 mm)
++++ 30980 voxels in the mask
+0.584105
+
+## Experiment 2 (smoothing FWHM = 3 mm)
++++ 30980 voxels in the mask
+0.491035
+
+# Interpretation
+The Dice coefficient is larger when the smoothing kernel is larger.
+This is because there is less noise when the data is more smoothed, resulting in higher reliability between test and retest sessions due to more robust signal.
```

Comment:
Yes, good. You had a mistake specifying the contrast, but you should get the same general finding regardless.